app/services/throttle.rb

```diff
@@ -4,63 +4,12 @@
 class Throttle
   attr_reader :throttle_type
 
-  THROTTLE_CONFIG = {
```

Comment:
What's the reason/context for refactoring this constant?

Reply:
I think I was having trouble with the tests and having multiple avenues to stub since it can be accessed through the constant and the `Throttle.attempt_window_in_minutes/1` method. It was a change I'd been considering for a bit and I swear there was a good reason to bring it into this PR in particular, but I'm struggling to remember the specifics 😓

Definitely open to improvements/reversion.